.github/workflows/inshpect.yml

```diff
@@ -0,0 +1,48 @@
+name: Inshepct
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  doc:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Install tools
+        run: |
+          set -x
```

Comment:
This should be removed before this is merged. You can keep it if needed for debugging for now though.